parcels/collection/collectionaos.py

```diff
@@ -842,14 +1131,15 @@ def merge(self, same_class=None):
 
         The function shall return the merged ParticleCollection.
         """
-        raise NotImplementedError
+        super().merge(other)
 
-    def split(self, indices=None):
+    def split(self, keys=None):
         """
         This function splits this collection into two disect equi-structured collections. The reason for it can, for
         example, be that the set exceeds a pre-defined maximum number of elements, which for performance reasons
         mandates a split.
 
+        TODO - RETHINK IF THAT IS STILL THE WAY TO GO:
```

Comment:
Also here; either fix or provide a bit more information for future users/developers